bin/bioformats.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
Nit: Can we add a docstring for the script here.  This is definitely a useful script to have because we will definitely run into more scenarios where we run a validation tool first then have to update the validation results.

We are fortunate here to not run into the issue of the tool container having Python, but for future iterations, it might be helpful to also have a "package-validate" step.